README.md

```diff
@@ -48,6 +54,25 @@ apt install polkadot
 
 ```
 
+#### Install using a user with elevated privileges (sudo)
+
+Run the following commands as a `non-root` user.
+
+```bash
+# Import the security@parity.io GPG key
+PARITY_SEC_KEY=9D4B2B6EB8F97156D19669A9FF0812D491B96798
+gpg --recv-keys --keyserver hkps://keys.mailvelope.com $PARITY_SEC_KEY
+sudo gpg --export $PARITY_SEC_KEY | sudo tee /usr/share/keyrings/parity.gpg
+# Add the Parity repository and update the package index
+echo 'deb [signed-by=/usr/share/keyrings/parity.gpg] https://releases.parity.io/deb release main' | sudo tee /etc/apt/sources.list.d/parity.list
+apt update
+# Install the `parity-keyring` package - This will ensure the GPG key
+# used by APT remains up-to-date
+apt install parity-keyring
+# Install polkadot
+apt install polkadot --version
```

Comment:
```suggestion
sudo apt install polkadot
```
see above

Reply:
Not sure if my suggestion got mashed up.
We def. do NOT want to run `apt install polkadot --version` but install polkadot first **and then** run `polkadot --version` to confirm the proper install.